docs/en/quick-answers/administering/share-assets-between-projects.md

```diff
@@ -9,20 +9,38 @@ lang: en
 
 It's not uncommon to have a project that's used as an Asset Library, containing assets that can be loaded into shots in other projects.
```

Comment:
While we are at it, we should update this to: 
It is common to have a project that is used as an Asset Library—containing assets that can be loaded into shots in other projects.